github-for-jira.sd.yml

```diff
@@ -129,8 +144,12 @@ environmentOverrides:
   prod:
     config:
       environmentVariables:
-        APP_URL: https://github-for-jira.prod.services.atlassian.com
+        APP_URL: https://github-for-jira.services.atlassian.com
+        WEBHOOK_PROXY_URL: https://github-for-jira.services.atlassian.com/github/events
         INSTANCE_NAME: github-for-jira-production
+        SENTRY_ENVIRONMENT: production
+        APP_ID: '125082'
+        GITHUB_CLIENT_ID: Iv1.234a3f06f2b64fbe
```

Comment:
This is configured against a new GitHub app I created as interim for until we're moving to the "real" GitHub app with the migration.